src/main/java/cn/nukkit/entity/monsters/EntitySkeleton.java

```diff
@@ -0,0 +1,92 @@
+package cn.nukkit.entity.monsters;
+
+import cn.nukkit.Player;
+import cn.nukkit.entity.EntityCreature;
+import cn.nukkit.event.entity.EntityDamageByEntityEvent;
+import cn.nukkit.item.Item;
+import cn.nukkit.item.ItemBow;
+import cn.nukkit.level.Level;
+import cn.nukkit.level.format.FullChunk;
+import cn.nukkit.nbt.tag.CompoundTag;
+import cn.nukkit.network.protocol.MobEquipmentPacket;
+import cn.nukkit.entity.mobutils.Drops;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class EntitySkeleton extends EntityCreature {
+
+    public static final int NETWORK_ID = 34;
+
+    public EntitySkeleton(FullChunk chunk, CompoundTag nbt) {
+        super(chunk, nbt);
+    }
+
+    @Override
+    public void initEntity() {
+        super.initEntity();
+
+        this.setMaxHealth(20);
+    }
+
+    @Override
+    public int getNetworkId() {
+        return NETWORK_ID;
+    }
+
+    @Override
+    public float getWidth() {
+        return 0.65f;
+    }
+
+    @Override
+    public float getHeight() {
+        return 1.8f;
+    }
+
+    @Override
+    public void spawnTo(Player player) {
+        super.spawnTo(player);
+
+        MobEquipmentPacket pk = new MobEquipmentPacket();
+        pk.eid = this.getId();
+        pk.item = new ItemBow();
+        pk.slot = 10;
+        pk.selectedSlot = 10;
+        player.dataPacket(pk);
+    }
+
+    @Override
+    public boolean entityBaseTick(int tickDiff) {
```

Comment:
You could make some kind of `BurningMob` abstract class for that behavior. Anyway, as I already said, it's early to implement it.